src/frontend/get-jira-configuration.ts

```diff
@@ -104,7 +112,7 @@ export default async (
 			)
 		);
 
-		const failedConnections = getFailedConnections(
+		const failedConnections = await getFailedConnections(
```

Comment:
does this mean `failedConnections` has always been an unresolved promise?

Reply:
No, it used to be a sync function which I changed to async for the `booleanFlag` call